File: ai-api-sdk/Java/ktAiApiClient/src/main/java/com/kt/ai/api/client/TtsClient.java
package com.kt.ai.api.client;

import kt.gigagenie.ai.api.TTS;
import org.json.JSONObject;

import java.io.*;

public class TtsClient {
    public static void main(String[] args) {
        TTS ttsClient = new TTS();

        String clientID = "client-id";
        String clientKey = "client-key";
        String clientSecret = "client-secret";

        ttsClient.setAuth(clientKey, clientID, clientSecret);

        String text = "안녕하세요 반갑습니다.";
        int pitch = 100;
        int speed = 100;
        int speaker = 1;
        int volume = 100;
        String language = "ko";
        String encoding = "mp3";
        int channel = 1;
        int sampleRate = 16000;
        String sampleFmt = "S16LE";

        try {
            JSONObject resultJson = ttsClient.requestTTS(text, pitch, speed, speaker, volume,
                    language, encoding, channel, sampleRate, sampleFmt);
            int statusCode = resultJson.optInt("statusCode");

            if(statusCode == 200) {

                byte[] audioData = (byte[]) resultJson.opt("audioData");

                String targetFilePath = "./src/main/resources/tts.mp3";
                System.out.println("targetFile:" + targetFilePath);

                FileOutputStream fos = null;
                try {
                    fos = new FileOutputStream(targetFilePath);
                } catch (FileNotFoundException e) {
                    e.printStackTrace();
                }

                try {
                    fos.write(audioData, 0, audioData.length);
                    fos.close();
                } catch (IOException e) {
                    e.printStackTrace();
                }
            }

        } catch (Exception e) {
            System.out.println("e >> " + e.getMessage());
        }
    }
}

File: ai-api-sdk/Python/v1.0/ktAiApiClient/tts_client.py
from ktAiApiSDK.tts import TTS
import base64
import json


def main():
    tts_client = TTS()

    client_id = "client-id"
    client_key = "client-key"
    client_secret = "client-secret"

    tts_client.setAuth(client_key, client_id, client_secret)

    text = "안녕하세요 반갑습니다"
    pitch = -1
    speed = -1
    speaker = 1
    volume = -1
    language = "ko"
    encoding = "mp3"
    channel = 1
    sample_rate = 16000
    sample_fmt = "S16LE"

    result_json = tts_client.requestTTS(text, pitch, speed, speaker, volume, language, encoding, channel, sample_rate, sample_fmt)
    result_json_dic = json.loads(result_json)
    print(result_json_dic)

    if result_json_dic["statusCode"] == 200:
        print(result_json_dic)
        try:
            with open('tts.mp3', 'wb') as f:
                f.write(base64.b64decode(result_json_dic["audioData"].encode('utf8')))
        except Exception as e:
            print("error = " + e)


if __name__ == "__main__":
	main()